packages/route-viewer-overlay/src/RouteViewerOverlay.story.js

```diff
@@ -1,34 +1,76 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import BaseMap from "@opentripplanner/base-map";
 import React from "react";
+import { GeoJSON } from "react-leaflet";
 
 import routeData from "../__mocks__/mock-route.json";
+import flexRouteData from "../__mocks__/mock-flex-route.json";
 import RouteViewerOverlay from ".";
 
 import "../../../node_modules/leaflet/dist/leaflet.css";
 
-const center = [45.543092, -122.671202];
+const PORTLAND = [45.543092, -122.671202];
+const POWDER_SPRINGS = [33.8595, -84.67483];
 const zoom = 11;
 
+// Hide some story args completely.
+const hiddenProp = {
+  table: { disable: true }
+};
+
 export default {
   title: "RouteViewerOverlay",
-  component: RouteViewerOverlay
+  component: RouteViewerOverlay,
+  argTypes: {
+    center: hiddenProp,
+    zoom: hiddenProp,
+    routeData: hiddenProp,
+    path: hiddenProp,
+    extraLayer: hiddenProp
+  }
```

Comment:
Sort attributes.

Reply:
Sorted in f13469bc7ec8b24af48c993101ec3ca3bb69afcb